content/docs/dvclive/user-guide/quickstart.md

```diff
@@ -0,0 +1,98 @@
+# Quickstart
+
+DVCLive is a simple Python library whose interface consists of three main
+methods.
+
+## Steps
+
+To get it up and running you just need to follow these steps:
+
+### 1. Initialize DVCLive
+
+```python
+import dvclive
+
+dvclive.init()
+```
+
+See [`dvclive.init()`] for details.
+
+### 2. Log metrics
+
+```python
+dvclive.log(metric_name, value)
+```
+
+See [`dvclive.log()`] for details.
+
+### 3. Increase the step number
+
+```python
+dvclive.next_step()
+```
+
+See [`dvclive.next_step()`] for details.
+
+## Putting all together
+
+Using the above steps, you can easily include DVCLive in your training code:
+
+```python
+# train.py
+
+import dvclive
+
+dvclive.init()
+
+for epoch in range(NUM_EPOCHS):
+    train_model(...)
+    metrics = evaluate_model(...)
+
+    for metric_name, value in metrics.items():
+        dvclive.log(metric_name, value)
+
+    dvclive.next_step()
+```
+
+## Outputs
+
+After you run your training code, you should see the following content in the
+project:
+
+```bash
+$ ls
+dvclive        train.py
+dvcelive.json
+```
+
+For each `{metric_name}`, DVCLive produces metrics _logs_ under
```

Comment:
@jorgeorpinel Is there a specified way to format this kind of text (`{metric_name}`, `<metric_name>`, etc.)?

Reply:
Hey sorry I didn't notice this earlier. I tuned out since 2 people were already reviewing. Please don't hesitate to reach out to me directly in these situations!

To answer the question, we typically use an example instead of placeholders. Something like `mymetric` in the code blocks, and then plain language in text e.g. "for each metric name, ..."

I'll do a copy edit PR of these docs ASAP.